@css/main_style.css

```diff
@@ -701,12 +704,6 @@ dl.tech-menu > dt {
 }
 
 .form-name-text-style {
-  display: flex;
-  min-width: 273px;
-  height: 18px;
-  flex-direction: column;
-  justify-content: center;
-  flex-shrink: 0;
   color: #172b4d;		/* #172B4D */
```

Comment:
- [ ] Kindly remove the commented-out code, it will make your code source clean and professional 🚰

Reply:
In relation to the inline comments I do not see how it can impair, in anyway, the readability of the code (neither in CSS files, nor in HTML files).

And although I am going to go through the additional required work to take them out, at this point, work is ongoing reformulating the whole set of CSS files (another active branch I am already working on), in preparation to insert the desktop version, so it will have to be seen as it will be then after the merge of the currently open pull requests.

But, for the moment, as requested, I will take them out although I can not agree with the fact that it makes the code less readable, understandable or less professional.